dev-requirements.txt

```diff
@@ -4,7 +4,7 @@
 
 flake8
 black
-jinja2
+jinja2<=3.0.3
```

Comment:
Temporary fix to make the tests pass as the latest jinja2 release from March 24 does not work with mkdocs
- [issue](https://github.com/mkdocs/mkdocs/issues/2794)
- [PR to fix](https://github.com/mkdocs/mkdocs/pull/2795)

Reply:
It looks like they've released a fix to mkdocs so we can instead unpin mkdocs or pin it to something higher

> MkDocs version 1.2.4 was just released with only a fix for this, as well as version 1.3.0 with more new features.

https://github.com/mkdocs/mkdocs/issues/2799#issuecomment-1079764329